src/bos/operators/base.py

```diff
@@ -55,6 +56,41 @@ class MissingSessionData(BaseOperatorException):
     desired state.
     """
 
+class ApiClients:
+    """
+    Context manager to provide API clients to BOS operators.
+    Essentially, it uses an ExitStack context manager to manage the API clients.
+    """
+
+    def __init__(self):
+        #self.bos = BOSClient()
+        #self.bss = BSSClient()
+        #self.cfs = CFSClient()
+        #self.hsm = HSMClient()
+        #self.ims = IMSClient()
+        #self.pcs = PCSClient()
```

Comment:
If not needed remove the comment code snippet

Reply:
It's only here for the purpose of the PR, to show what is coming.